core/src/test/java/org/elasticsearch/gateway/RecoveryFromGatewayIT.java

```diff
@@ -383,7 +384,7 @@ public void doAfterNodes(int numNodes, Client client) throws Exception {
 
         ClusterState state = client().admin().cluster().prepareState().execute().actionGet().getState();
         assertThat(state.metaData().index("test").mapping("type2"), notNullValue());
-        assertThat(state.metaData().templates().get("template_1").template(), equalTo("te*"));
+        assertThat(state.metaData().templates().get("template_1").template(), equalTo(Collections.singletonList("te*")));
```

Comment:
Personally I do a static import for `singletonList` but I know lots of folks don't. If you want to keep it as is that is fine with me, or if you want to do static import that is fine with me.